addon/templates/components/welcome-page.hbs

```diff
@@ -1,17 +1,19 @@
-<div class="welcome-box cf" data-ember-version="{{emberVersion}}">
-  <div class="tomster">
-    <img src="/ember-welcome-page/images/construction.png" alt="Under construction">
-  </div>
-  <div class="welcome">
-    <h2 id="title">Congratulations, you made it!</h2>
+<div id="welcome-page" class="cf" data-ember-version="{{emberVersion}}">
```

Comment:
Let's get something longer and less likely to conflict with apps (I have had components in my apps named welcome-page before)...